source/install/self-managed-changelog.md

```diff
@@ -4,13 +4,143 @@
 Also see [changelog in progress](https://bit.ly/2nK3cVf) for the next release.
 
 Lastest Mattermost Releases:
+- [Release v6.1 - Feature Release](#release-v6.1-feature-release)
 - [Release v6.0 - Feature Release](#release-v6.0-feature-release)
 - [Release v5.39 - Quality Release](#release-v5-39-quality-release)
 - [Release v5.38 - Feature Release](#release-v5-38-feature-release)
 - [Release v5.37 - Extended Support Release](#release-v5-37-extended-support-release)
-- [Release v5.36 - Feature Release](#release-v5-36-feature-release)
-- [Release v5.35 - Feature Release](#release-v5-35-feature-release)
 
+## Release v6.1 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)
+
+**Release Day: 2021-11-16**
+
+### Important Upgrade Notes
+ - The Bleve index has been updated to use the scorch index type. This new default index type has some improvements on efficiency, which means that the indexes use significantly less disk space. To use this new type of index, after upgrading the server version, run a purge operation and then a reindex from the Bleve section of the System Console. Bleve is still compatible with the old indexes, so the currently indexed data will work fine if the purge and reindex is not run.
+ - A composite index has been added to the jobs table for better query performance. For some customers with large jobs table, this can take a long time, so we recommend adding the index during off-hours, and then running the migration.
+   - For PostgreSQL: ``create index concurrently idx_jobs_status_type on jobs (status,type);``
+   - For MySQL: ``create index idx_jobs_status_type on Jobs (Status,Type);``
+
+**IMPORTANT:** If you upgrade from a release earlier than v6.0, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
+
+### Highlights
+
+#### Timed Do Not Disturb
+ - Added the ability to disable all notifications for a specified period of time to avoid distractions, without losing important messages when you're back.
+
+#### Cross-team Recent Mentions
+ - Recent mentions and saved posts now show across all teams.
```

Comment:
Searches as well, correct, given the changes to POST /api/v4/posts/search documented later on this page?

Reply:
@larkox Can you help confirm?

Reply:
Searches are limited to the current team by the user agent (webapp/mobile). There are plans to include that in the future, but not yet.

Reply:
cc @esethna